Autocoders/Python/src/fprime_ac/generators/templates/component/hpp.tmpl

```diff
@@ -580,7 +580,7 @@ $emit_port_params(8, $params)
       #set $inst = $inst + 1
     #end for
   #end for
-  #for $ids, $name, $type, $osets, $osavs, $size, $default, $comment, $typeinfo in $parameters:
+  #for $ids, $name, $type, $osets, $osaves, $size, $default, $comment, $typeinfo in $parameters:
```

Comment:
I can't figure out what these things are.

Are they all the same? Are they different?

Some have `$osavs`, some `$osave`

Reply:
This is a template file. `$osavs` is used to represent opcode save routines when parameters are specified.

Reply:
Is `$osavs` the same as `$osave` below: https://github.com/nasa/fprime/pull/437#discussion_r587125447?

Reply:
This file is ok.  You caught the references on 594 and 595